src/android/java/io/jxcore/node/BtToRequestSocket.java

```diff
@@ -165,82 +150,89 @@ public void StreamCopyError(StreamCopyingThread who, String error) {
         }
     }
 
-    public int GetLocalHostPort() {
+    private  int GetLocalHostPort() {
         return mHTTPPort;
     }
 
-    public String GetLocalHostAddress() {
-        String ret = "";
-        if(localHostSocket != null && localHostSocket.getInetAddress() != null){
-            ret = localHostSocket.getInetAddress().toString();
-        }
-        return ret;
+    private String GetLocalHostAddressAsString() {
+        return localHostSocket == null || localHostSocket.getInetAddress() == null ? null : localHostSocket.getInetAddress().toString();
     }
 
     public void Stop() {
         mStopped = true;
```

Comment:
Fair point on run. But in that case, what purpose is mStopped? It gets set but near as I can tell it literally never gets checked. So why not just remove it?